rpc-rs/src/provider_main.rs

```diff
@@ -215,3 +256,126 @@ pub fn load_host_data() -> Result<HostData, RpcError> {
     })?;
     Ok(host_data)
 }
+
+#[cfg(feature = "otel")]
+const TRACING_PATH: &str = "/v1/traces";
```

Comment:
From https://opentelemetry.io/docs/reference/specification/protocol/otlp/#otlphttp-request

> The default URL path for requests that carry trace data is /v1/traces (for example the full URL when connecting to “example.com” server will be https://example.com/v1/traces). The request body is a Protobuf-encoded ExportTraceServiceRequest message.

It says that non-default paths MAY be used by a server and client. That seems to imply to me that the client and server agree to it beforehand. So we may need to offer this as a configuration value in the future, but this is good enough for now because it adheres to spec